src/corehost/src/servicing_index.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "trace.h"
+#include "servicing_index.h"
+
+servicing_index_t::servicing_index_t(const arguments_t& args)
+{
+    m_patch_root = args.svc_dir;
```

Comment:
Don't know if you're planning a separate formatting pass, but the formatting is a bit off here :)